src/webapp/components/data-submissions-history/DataSubmissionsHistoryContent.tsx

```diff
@@ -1,32 +1,23 @@
 import { DataSubmissionsTable } from "./DataSubmissionsTable";
-import { useState } from "react";
-import { useLocation } from "react-router-dom";
 import { useAppContext } from "../../contexts/app-context";
 import { useGlassDataSubmissionsByModuleAndOU } from "../../hooks/useGlassDataSubmissionsByModuleAndOU";
 import { ContentLoader } from "../content-loader/ContentLoader";
+import { useCurrentOrgUnitContext } from "../../contexts/current-orgUnit-context";
+import { useCurrentModuleContext } from "../../contexts/current-module-context";
 
-interface DataSubmissionsHistoryContentProps {
-    moduleName: string;
-    moduleId: string;
-}
-
-export const DataSubmissionsHistoryContent: React.FC<DataSubmissionsHistoryContentProps> = ({
-    moduleId,
-    moduleName,
-}) => {
+export const DataSubmissionsHistoryContent: React.FC = () => {
     const { compositionRoot } = useAppContext();
-    const location = useLocation();
-    const queryParameters = new URLSearchParams(location.search);
-    //TO DO : The orgUnit should come from a global context which is yet to be implemented.
-    const orgUnitVal = queryParameters.get("orgUnit");
-    const [orgUnit] = useState(orgUnitVal === null ? "" : orgUnitVal);
-    const dataSubmissions = useGlassDataSubmissionsByModuleAndOU(compositionRoot, moduleId, orgUnit);
+    const { currentOrgUnitAccess } = useCurrentOrgUnitContext();
+    const { currentModuleAccess } = useCurrentModuleContext();
+    const dataSubmissions = useGlassDataSubmissionsByModuleAndOU(
+        compositionRoot,
```

Comment:
Typically we don't pass the `compositionRoot` to a custom hook, as the hook itself can get it with `useAppContext`

Reply:
Addressed in https://github.com/EyeSeeTea/glass-dev/commit/c664c49dc8abcbeb0baeceac61d29b7d7275cb08